script/codesign/import-testing-cert-ci.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+KEY_CHAIN=mac-build.keychain
+KEYCHAIN_PASSWORD=unsafe_keychain_pass
+security create-keychain -p $KEYCHAIN_PASSWORD $KEY_CHAIN
+# Make the keychain the default so identities are found
+security default-keychain -s $KEY_CHAIN
+# Unlock the keychain
+security unlock-keychain -p $KEYCHAIN_PASSWORD $KEY_CHAIN
+# Set keychain locking timeout to 3600 seconds
+security set-keychain-settings -t 3600 -u $KEY_CHAIN
```

Comment:
why 3600?

Reply:
It's a substantially long enough period of time (an hour) that there's no way the keychain will relock before the tests run on the CI box.

Reply:
Yep, I'm sure our tests will never ever take an hour to run :laughcry: